MSSQL Integration: Adds query_config_path to allow for custom metrics through custom exporter config file (#5768)
grafana/agent: docs/sources/static/configuration/integrations/mssql-config.md

  # The timeout for scraping metrics from the mssql instance.
  [timeout: <duration> | default = "10s"]

  # The file path for a YAML config file which allows for custom queries/metrics
  # for the mssql instance
  [query_config_path: <string>]

```

## Custom metrics
It is possible to retrieve custom prometheus metrics for a mssql instance using the optional `query_config_path` parameter.

This parameter should point to a YAML config file defined [here](https://github.com/burningalchemist/sql_exporter#collectors). If it does, it will use the new config to query your mssql instance and create whatever metrics are defined.

> If you want additional metrics on top of the default provided ones, the default config should be used as a base.

Reviewer: ```suggestion
If you want additional metrics on top of the default metrics, the default configuration must be used as a base.
```
